### DISCUSSION TITLE
Wrong file on project page

### DISCUSSION THREAD

**Discussion by Carrera124** [pybricks/support#1744] [General]
Hello, I am sorry, but I couldn't find any other communication method to report this to the site owner(s):

I was digging through the project pages, and found that the download links on this page contain at least 1 wrong file:
https://pybricks.com/project/technic-42146-powered-up-remote/
Header says "powered up remote", but the download for the base code file delivers an Xbox Controller file.
However, I didn't check /compare the diagrams and sourcecodes, there I cannot say if it is only the download file or also the diagrams and/or source code.

**Comment by dlech** [comment]
Thanks for reporting the issue. It should be fixed now.

https://github.com/pybricks/pybricks.github.io/commit/4748cfe3890bd450263e9babfea9bf89d81b67ac

**Reply by laurensvalk** [reply]
Thank you. That first note in that commit can be removed. That is indeed accidentally copied info about compatible Xbox controllers but isn't relevant for the powered up remote.
